tools/paths.py

```diff
@@ -76,12 +76,6 @@
 VODAFONE_LIBRARY = join(NET_LIBRARIES, "VodafoneUSBModem")
 UBLOX_LIBRARY = join(NET_LIBRARIES, "UbloxUSBModem")
 
-# FS
-FS_PATH = join(LIB_DIR, "fs")
-FAT_FS = join(FS_PATH, "fat")
-SD_FS = join(FS_PATH, "sd")
-FS_LIBRARY = join(BUILD_DIR, "fat")
```

Comment:
Yes. I discussed this with @sg-  and @bridadan yesterday and this was their recommendation.